src/components/Reports/ReportEdit.tsx

```diff
@@ -186,23 +186,23 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
     }
   };
 
-  const updateSeedbank = (seedbankIndex: number, seedbankField: string, value: any) => {
-    if (report && report.seedBanks) {
+  const updateLocation = (index: number, field: string, value: any, location: 'seedBanks' | 'nurseries') => {
+    if (report && report[location]) {
       setReport(
         produce((draft) => {
           // @ts-ignore
-          draft.seedBanks[seedbankIndex][seedbankField] = value;
+          draft[location][index][seedbankField] = value;
```

Comment:
`seedbankField` should be `field` here.

Reply:
Oops, thanks, 👍 
I don't know how this worked in my testing